core-container/entry.sh

```diff
@@ -218,6 +222,41 @@ function run_placement_translator() {
 }
 
 
+function run_dex() {
+    echo "--< Starting dex in background>--"
+    dex serve /home/kubestellar/dex/dex-config-dev.yaml &
+}
+
+
+function run_kb_backend() {
+    echo "--< Starting kube-bind example-backend in background >--"
```

Comment:
don't we want to do this when we create espw ?

Reply:
Yes, moved into `function ensure_espw()`